torch/_inductor/pattern_matcher.py

```diff
@@ -2100,15 +2101,17 @@ def fwd_only(
     args: Sequence[Any],
     *,
     run_functional_passes: bool = True,
+    should_preserve_node_meta: bool = False,
```

Comment:
Looks reasonable - cc @bdhirsh, @zou3519 any thoughts on if this should be on more generally ?